tests/test_python_library.py

```diff
@@ -126,3 +128,37 @@ def test_split_system_tests():
         with open(templated_files / ".kokoro/presubmit/system-3.8.cfg", "r") as f:
             contents = f.read()
             assert "system-3.8" in contents
+
+
+@pytest.mark.parametrize(
+    "fixtures_dir",
+    [
+        Path(__file__).parent / "fixtures/python_library",  # just setup.py
+        Path(__file__).parent
+        / "fixtures/python_library_w_version_py",  # has google/cloud/texttospeech/version.py
+    ],
+)
+def test_configure_previous_major_version_branches(fixtures_dir):
+    with util.copied_fixtures_dir(fixtures_dir):
+        t = templates.Templates(PYTHON_LIBRARY)
+        result = t.render(".github/release-please.yml")
+        os.makedirs(".github")
+        shutil.copy(result, Path(".github/release-please.yml"))
+
+        python.configure_previous_major_version_branches()
+        release_please_yml = Path(".github/release-please.yml").read_text()
+
+        assert (
+            release_please_yml
+            == """releaseType: python
+handleGHRelease: true
+  # NOTE: this section is generated by synthtool.languages.python
+  # See https://github.com/googleapis/synthtool/blob/master/synthtool/languages/python.py
+  branches:
+  - branch: v1
+    handleGHRelease: true
+    releaseType: python
+  - branch: v0
+    handleGHRelease: true
+    releaseType: python"""
```

Comment:
I think this indentation is wrong. `branches` should not have leading spaces

Reply:
Fixed!